python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -23,7 +24,17 @@
 
 class RankingConstants:
     NEW_RANK = "new_rank"
+    NEW_MRR = "new_MRR"
+    OLD_MRR = "old_MRR"
+    OLD_RANKNMF = "old_RankNMF"
+    NEW_RANKNMF = "new_RANKNMF"
```

Comment:
@mohazahran I think we should add these later (or in a different PR).

Reply:
ping

Reply:
there are just constant definitions. but can remove them from now.